From e1e58218c9f15b1b3654fe0bc9aacdaac05a37a5 Mon Sep 17 00:00:00 2001
From: "Christian@p620" <Christian@p620>
Date: Mon, 26 Sep 2022 12:09:17 +0200
Subject: [PATCH 109/151] Disable AVB port in DT as this is currently not
 functional. This prevents messages and eth3 configuration in case of
 connecting Ethernet cable

---
 arch/arm64/boot/dts/renesas/r8a779f0-vc4-ctrl-domain.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a779f0-vc4-ctrl-domain.dts b/arch/arm64/boot/dts/renesas/r8a779f0-vc4-ctrl-domain.dts
index 75bb2a808dae..a68632b04990 100644
--- a/arch/arm64/boot/dts/renesas/r8a779f0-vc4-ctrl-domain.dts
+++ b/arch/arm64/boot/dts/renesas/r8a779f0-vc4-ctrl-domain.dts
@@ -122,7 +122,7 @@
 };
 
 &etnb0 {
-	status = "okay";
+	//status = "okay";
 
 	pinctrl-0 = <&etnb0_pins>;
 	pinctrl-names = "default";
-- 
2.17.1

